### PR TITLE
fix: users authenticate once with local auth

### PR DESCRIPTION
**Pull request by mcgarj**
# DCMAW-9700: User is required to authenticate with local auth only once to log in

This PR now stores the `idToken` and `accessToken` in a single object: `StoredKeys`. It also creates a service, `StoredKeyService` to handle saving and retrieving the object from the secure store. On saving, the keys are encoded to a `base64String` and saved. When fetching them, they are decoded to the `StoredKeys` object, which can then be accessed in the code where needed with `StoredKeys.idToken` etc

# Checklist

## Before raising your pull request:
- [x] Commit messages that conform to conventional commit messages
- [x] Ran the app locally ensuring it builds 
- [x] Ran the tests locally ensuring they pass on Build
- [x] Pull request has a clear title with a short description about the feature or update
- [x] Created a `draft` pull request if it is not yet ready for review

## Before your pull request can be reviewed:
- [x] Met all of the acceptance criteria specified in the user story on Jira
- [x] Reviewed your own code to ensure you are following the style guidelines
- [x] Ran the app and tested the feature on a range of device sizes
      Please include iPod Touch, iPhone SE and iPhone 11 as a minimum.
- [x] Written Unit and Integration tests if needed

- [x] Met all accessibility requirements?
    - [x] Checked dynamic type sizes are applied
    - [x] Checked VoiceOver can navigate your new code
    - [x] Checked a user can navigate only using a keyboard around your new code 

## Before merging your pull request:
- [x] Ensure that the code coverage and SonarCloud checks have passed
- [x] Actioned and resolved all comments, reaching out to reviewers for clarifications if necessary.
- [x] Ran the app to ensure that no regressions have been caused by changes during code review.

### CODE REVIEW

**Comment by obinns-dd** [comment]
Looking really good- are we able to chase down the last 10.8% of untested code? 😅 
image

**Review by Magdaline-A — APPROVED** [review]

File: Tests/UnitTests/Mocks/Sessions+Services/MockStoredKeyService.swift
Comment (obinns-dd) [inline Tests/UnitTests/Mocks/Sessions+Services/MockStoredKeyService.swift:1]: I think we just missed renaming the file here when we renamed the class 🙂

Comment (mcgarj) [reply Tests/UnitTests/Mocks/Sessions+Services/MockStoredKeyService.swift:1]: Good catch!

**Review by obinns-dd — APPROVED** [review]

**Comment by obinns-dd** [comment]
Really great work! Looks all good to me! 😄 🎉

**Pull request merged**